packages/react-native-reanimated/apple/reanimated/apple/Fabric/REAInitializerRCTFabricSurface.h

```diff
@@ -5,7 +5,7 @@
 
 @interface REAInitializerRCTFabricSurface : RCTFabricSurface
 
-@property REAModule *reaModule;
+@property __weak REAModule *reaModule;
```

Comment:
Since surface is `static`, when destroying the modules etc, `REAModule` would not be deallocated since `REAInitializerRCTFabricSurface` still held reference to it.